### PR TITLE
Rs 2021 12 05

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
 <Page3>
-Sand and debris billow up into the air.
-The wind whipped up a cloud of dust that sweeps the area.
-The river of flame that had swept across the downtown area was eradicated by that single blow.
+Earth and debris billow up into the air.
+The wind whipped up a cloud of dust that scoured the area.
+The river of flame that had been flowing over the downtown area was halted by that single blow.
 <Page4>
-The policemen that surround Vlov in the distance are blown away, incapacitating most of them. They are divided into two groups: those who became unconscious after suffering a head injury and those who lose their will to fight.
+The policemen that had been encircling Vlov are blown away, and most of them are incapacitated in the process. Half of them have been knocked unconscious by blows to the head, and the rest have simply lost their will to fight.
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt:12]: incapacitating most of them in the process.
(I feel like the 'and' slows the flow of the sentence down and is already used in the next one.)

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
 It's not surprising.
 <Page5>
 "In-Incre..."
-After the dust had settled, the result was in plain view.//to be rechecked "this is the result" sounds bad apparently, なにしろ
-Arcueid's rapid descent not only shattered the area around Vlov, but also a large portion of the downtown area at the north gate.
+Once the dust has settled, the extent of the destruction is plain to see. // これである reified for flow
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt:16]: had
(the destruction can only be seen once the dust has finished settling)

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
+Arcueid's rapid descent not only shattered the area around Vlov, but also collapsed a significant portion of the business district around the north gate.
 <Page6>
 So that's what she meant by 'dropping them underground'.
-The land had sunk over 40 meters.
+The land has sunk over 40 meters.
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt:20]: had
(again, the land had already sunk. he's viewing the aftermath now)

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt [hunk 4]
```diff
@@ -35,67 +35,67 @@ Did that flame come from Vlov?
...
-That's because if Arcueid is to be defeated by Vlov, the only choice I would have left is to run away.
+That's because if Arcueid were to be defeated by Vlov, the only choice I would have left is to run away.
 <Page20>
 "Hah――――――hah."
 I reach the elevator.
 I head inside and press the button for the first floor.
 There's no more time for hesitation.
-This isn't the time to hesitate in the first place.
-I take a deep breath. I check my condition. I can move properly at the very least.
+There was never any any time to hesitate in the first place.
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt:78]: duplicate any

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_6_1.txt [hunk 5]
```diff
@@ -17,11 +17,11 @@ It isn't an intentional act, but an act of reflex in order to protect my life.
...
 There is oxygen here. I already established from the fight in the hotel that these blue flames are simply a curse and not a combustion of matter.
 <Page5>
-But that doesn't change the fact that it's sweltering in here. If I just breathe normally, the hot air will burn my lungs to a crisp.
+But that doesn't change the fact that it's sweltering down here. If I so much as breathe, the hot air will burn my lungs to a crisp.
 <Page6>
-A vampire in white and a vampire in a black cloak can be seen ahead.//to be rechecked "A vampire in a black mantle, cloak"
+Ahead of me, I can see one vampire in white, and another in a black cloak.
 The clash of sword with claw.
-Even in this heat, Arcueid is fighting Vlov with everything she's got without faltering.
+Even in this heat, Arcueid is laying into Vlov with an unrelenting offence.
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_6_1.txt:24]: offense is american

**Review by yk1512 — CHANGES_REQUESTED** [review]

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt:12]: Hmm, that doesn't quite read right to me:
> The policemen that had been encircling Vlov are blown away, incapacitating most of them in the process.

It feels like it's missing a reference to what is _doing_ the incapacitating. Maybe
> The blast blows away the policemen that had been encircling Vlov, incapacitating most of them in the process.

?

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt:16]: On this and the next one, is that definitely the correct tense? I assume he is talking about things in the moment, so if something happened just a second ago I feel like `has` is correct? Would `had` not be if he were recounting this story at a later date?

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt (hunk 1 above)
Comment (yk1512) [reply script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt:12]: I assumed that being blown away was what incapacitated them.

Your new sentence with the blast works though.

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt (hunk 2 above)
Comment (yk1512) [reply script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt:16]: Tried reading it several times but I still stand by my suggestion. Will have to wait for a third opinion

Comment (rschlaikjer) [reply script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt:16]: On this and the next one, is that definitely the correct tense? I assume he is talking about things in the moment, so if something happened just a second ago I feel like `has` is correct? Would `had` not be if he were recounting this story at a later date?

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt:12]: It is, but for some reason that formulation of the sentence feels like it skips something to me /shrug
Will use the second one.

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt (hunk 2 above)
Comment (Avuxo) [reply script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt:16]: I agree with Ross but propose "Once the dust settles"

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt (hunk 3 above)
Comment (Avuxo) [reply script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt:20]: i say drop the word entirely.

**Review by Avuxo — APPROVED** [review]

**Pull request merged**